mappings/net/minecraft/resource/ResourcePack.mapping

```diff
@@ -1,12 +1,19 @@
 CLASS wh net/minecraft/resource/ResourcePack
 	METHOD a getName ()Ljava/lang/String;
 	METHOD a getNamespaces (Lwi;)Ljava/util/Set;
+		ARG 1 type
 	METHOD a findResources (Lwi;Ljava/lang/String;ILjava/util/function/Predicate;)Ljava/util/Collection;
 		ARG 1 type
 		ARG 2 namespace
+		ARG 3 maxDepth
+		ARG 4 fileNameFilter
```

Comment:
`ResourcePack` is not file-specific, it could be implemented using something other than files. `identifierFilter` is better.

Reply:
This only filters the path of the id, so I will call it `pathFilter`

Reply:
Oh, never mind then. `fileNameFilter` is better. You're right, it includes the extension so it's not just the ID.

Reply:
Will say `pathFilter` because in resource managers, the raw path stored in ids still include the file suffix. E.g. when game scans for resource type `advancements` with suffix `.json`, `minecraft:advancements/story/root.json` will be returned by the resource manager as the id, which the advancement loader will then truncate to `minecraft:story/root`.

Reply:
It seems like `fileNameFilter` was originally correct: `filter.test(path2.getFileName().toString()`